### PR TITLE
Revert "Preserve quotedness for Identifier when formatting Query"

### PR DESCRIPTION
**Pull request by varungajjala**
This reverts commit d8323f56f8b2828163a6161311115257beda2cc8.

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by varungajjala** [comment]
cc @v-jizhang 

This is failing some of the tests. Please look at the stack trace below.

```
java.lang.AssertionError: expected [CREATE VIEW hive_view.hive_test.hive AS
SELECT *
FROM
  orders] but found [CREATE VIEW hive.hive_test.hive_view AS
SELECT *
FROM
  orders]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:179)
	at com.facebook.presto.spark.TestPrestoSparkQueryRunner.testCreateDropView(TestPrestoSparkQueryRunner.java:993)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Review by highker — APPROVED** [review]
would be good to have @rschlussel to take a look as well

File: presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java [hunk 1]
```diff
@@ -595,16 +596,6 @@ protected Node visitShowCreateFunction(ShowCreateFunction node, Void context)
                     ordering(ascending("argument_types")));
         }
 
-        private QualifiedName getQualifiedName(ShowCreate node, QualifiedObjectName objectName)
```
Comment (rschlussel) [inline presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java:598]: @v-jizhang looks like this is getting the parts for the name backwards.

**Review by rschlussel — APPROVED** [review]
This issue isn't just Presto on Spark, so we're lucky presto on spark had that round trip test.  We should add better basic tests for show Create View and show Create Table to regular Presto.

File: presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java (hunk 1 above)
Comment (v-jizhang) [reply presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java:598]: Trino still has the same code
https://github.com/trinodb/trino/blob/99ef4ec162b3b099762e4d6727ee3db641e1445c/core/trino-main/src/main/java/io/trino/sql/rewrite/ShowQueriesRewrite.java#L593

This PR was requested by some users, I'll do a fix and resubmit.

**Pull request merged**